tensorflow/core/profiler/internal/gpu/BUILD

```diff
@@ -138,6 +140,24 @@ tf_cuda_library(
     ],
 )
 
+tf_cuda_library(
+    name = "rocm_tracer",
+    srcs = if_rocm(["rocm_tracer.cc"]),
+    hdrs = if_rocm(["rocm_tracer.h"]),
+    copts = tf_copts(),
```

Comment:
```suggestion
    copts = tf_profiler_copts() + tf_copts(),
```

Reply:
Just to keep this consistent with the above.

Reply:
done